pkg/gce-pd-csi-driver/node.go

```diff
@@ -221,6 +225,11 @@ func (ns *GCENodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStage
 		return nil, status.Error(codes.InvalidArgument, "NodeStageVolume Volume Capability must be provided")
 	}
 
+	if _, alreadyExists := ns.volumes.LoadOrStore(volumeID, true); alreadyExists {
+		return nil, status.Error(codes.AlreadyExists, "An operation with the given Volume ID already exists")
```

Comment:
the correct error code according to the spec is actually: `10 ABORTED`

https://github.com/container-storage-interface/spec/blob/master/spec.md